pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/MinionClusterIntegrationTest.java

```diff
@@ -108,56 +90,76 @@ public void testStopAndResumeJobQueue()
     Map<String, TaskState> taskStates = _pinotHelixTaskResourceManager.getTaskStates(TestTaskGenerator.TASK_TYPE);
     Assert.assertEquals(taskStates.size(), 4);
     for (TaskState taskState : taskStates.values()) {
-      Assert.assertTrue(taskState.equals(TaskState.IN_PROGRESS));
+      Assert.assertEquals(taskState, TaskState.IN_PROGRESS);
     }
 
-    // Stop the job queue
+    // Stop the task queue
     _pinotHelixTaskResourceManager.stopTaskQueue(TestTaskGenerator.TASK_TYPE);
 
-    // Wait at most 60 seconds for all tasks stopped
+    // Wait at most 60 seconds for all STOPPED
     endTime = System.currentTimeMillis() + 60_000L;
     int stoppedTaskCount = 0;
     while ((System.currentTimeMillis() < endTime) && (stoppedTaskCount != 4)) {
       Thread.sleep(100L);
       stoppedTaskCount = 0;
       taskStates = _pinotHelixTaskResourceManager.getTaskStates(TestTaskGenerator.TASK_TYPE);
       for (TaskState taskState : taskStates.values()) {
-        if (taskState.equals(TaskState.STOPPED)) {
+        if (taskState == TaskState.STOPPED) {
           stoppedTaskCount++;
         }
       }
     }
     Assert.assertEquals(stoppedTaskCount, 4);
```

Comment:
Add a message to this assert saying that all tasks did not stop within the expected time, to make it easier to analyze the assertion failure. Do it for all assertions that are time-based in this test.